common/reactive/pom.xml

```diff
@@ -67,13 +67,8 @@
             <artifactId>helidon-common-mapper</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.hamcrest</groupId>
-            <artifactId>hamcrest-all</artifactId>
+            <groupId>io.helidon.microprofile.bundles</groupId>
```

Comment:
Please do not use Helidon microprofile libraries in SE modules

Reply:
fixed